scripts/linux/trcE2ETest.sh

```diff
@@ -866,10 +867,12 @@ quickstart_working_folder="$working_folder/quickstart"
 image_architecture_label=$(get_image_architecture_label)
 optimize_for_performance=true
 log_upload_enabled=true
+log_rotation_max_file="50"
```

Comment:
This is my justification for this number:

~15 modules * 10MB per file * 50 files = 7.5 GB total logs